projects/dsp-ui/src/lib/action/components/select-project/select-project.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="dsp-select-project-component">
+    <form class="form-content">
+        <mat-form-field class="large-field select-project">
+
+            <input
+                matInput
+                placeholder="Select project"
+                [matAutocomplete]="project"
+                [formControl]="selectProjectForm.controls['projectName']">
+
+            <button
+                mat-icon-button
+                matSuffix
+                *ngIf="selectProjectForm.controls['projectName'].value"
```

Comment:
Instead of `selectProjectForm.controls['projectName']` you could assign the instance of `FormControl` to a member of `SelectProjectComponent` and access it like this, see 

https://github.com/dasch-swiss/dsp-ui-lib/blob/fbb655e3441502edd9270d5437fa0253449250e0/projects/dsp-ui/src/lib/viewer/values/int-value/int-value.component.ts#L42-L58

Reply:
then from the template you can directly access the class's member `valueFormControl`